### PR TITLE
Adds agent card logging

### PR DESCRIPTION
**Pull request by 81Denton**
This PR adds agent card logging (forging/resetting) and replaces ugly single letter var names with more sensible ones.

🆑 Denton
admin: Added agent card logging (forging/resetting) to game.log
/🆑

### CODE REVIEW

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -421,16 +421,17 @@ update_label()
...
+			var/target_occupation = copytext(sanitize(input(user, "What occupation would you like to put on this card?\nNote: This will not grant any access levels other than Maintenance.", "Agent card job assignment", assignment ? assignment : "Assistant") as text | null),1,MAX_MESSAGE_LEN)
+			if(!target_occupation)
 				registered_name = ""
 				return
-			assignment = u
+			assignment = target_occupation
 			update_label()
 			forged = TRUE
 			to_chat(user, "<span class='notice'>You successfully forge the ID card.</span>")
+			log_game("[key_name(user)] has forged [src] with name \"[registered_name]\" and occupation \"[assignment]\".")
```
Comment (kriskog) [inline code/game/objects/items/cards_ids.dm]: Name changes based on registered_name and assignment, so [src] is the new card name here.

File: code/game/objects/items/cards_ids.dm [hunk 2]
```diff
@@ -451,6 +452,7 @@ update_label()
 			update_label()
 			forged = FALSE
 			to_chat(user, "<span class='notice'>You successfully reset the ID card.</span>")
+			log_game("[key_name(user)] has reset [src] to default name \"[registered_name]\" and occupation \"[assignment]\".")
```
Comment (kriskog) [inline code/game/objects/items/cards_ids.dm]: [src] is now "identification card" and the other two are null.

**Review by kriskog — CHANGES_REQUESTED** [review]

File: code/game/objects/items/cards_ids.dm (hunk 2 above)
Comment (81Denton) [reply code/game/objects/items/cards_ids.dm]: I'll uhhhh actually read the code now and fix that

File: code/game/objects/items/cards_ids.dm (hunk 1 above)
Comment (81Denton) [reply code/game/objects/items/cards_ids.dm]: Same here

**Comment by 81Denton** [comment]
Alright, I replaced it with [initial(name)].

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**